### ISSUE TITLE
Up Arrow Key Is Not Scrolling

### ISSUE THREAD

**Issue by kodaichodai** [caido/caido#1544]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Bug description

One user reported that the screen doesn't scroll as expected when a row is selected in HTTP History and they try to scroll the pane with the up arrow key.

Check: 
https://youtu.be/3bLy7wmx4js

I've tried to reproduce this but am unable to see this behavior as of 5/3/25, so I'm creating this issue to accumulate all the cases where the same issue occurs. One possibility is the size of the display or scale and resolution. The user in question has reported that his setting is as follows:

resolution: 1920 × 1080
scale: 150%

https://discord.com/channels/843915806748180492/897575154526330960/1368122097918017588

### Environment & setup

OS: Windows
OS version: 11 Pro
Caido client: Desktop
Caido version: 0.47.1

**Comment by Sytten** [comment]
This might be fixed in nightly
